src/register/constants/organisationAddress.ts

```diff
@@ -27,69 +27,79 @@ export const organisationAddress = {
     ],
     groups: [
       {
-        input: {
-          label: {
-            text: 'Building and street',
-            classes: 'govuk-label--m',
+        fieldset: [
+          {
+            legend: {
+              text: 'What\'s the address of your main office?',
+              classes: 'govuk-fieldset__legend--xl'
+            }
           },
-          validators: ['required'],
-          validationError: {
-            value: 'Enter Building and street',
-            controlId: 'officeAddressOne',
+          {
+            input: {
+              label: {
+                text: 'Building and street',
+                classes: 'govuk-label--m',
+              },
+              validators: ['required'],
+              validationError: {
+                value: 'Enter Building and street',
+                controlId: 'officeAddressOne',
+              },
+              control: 'officeAddressOne',
+              classes: '',
+            },
           },
-          control: 'officeAddressOne',
-          classes: '',
-        },
-      },
-      {
-        input: {
-          validationError: {
-            value: 'Enter the length of hearing in minutes, for example "20"',
-            controlId: 'officeAddressTwo',
-          },
-          control: 'officeAddressTwo',
-          classes: '',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'Town or city',
-            classes: 'govuk-label--m',
+          {
+            input: {
+              validationError: {
+                value: 'Enter the length of hearing in minutes, for example "20"',
+                controlId: 'officeAddressTwo',
```

Comment:
The `value` and `controlId` don't seem to match...?

Reply:
rectified